components/user/UserProfile.js

```diff
@@ -68,7 +68,7 @@ function UserProfile({ BASE_URL, data }) {
           </div>
         </div>
       </div>
-      <div className="flex justify-center my-4 text-center">
+      <div className="justify-center my-4 text-center">
```

Comment:
This still shows multiple lines from the textarea as 1 line

Reply:
I have committed changes for this issue